src/fides/api/util/tcf/tc_mobile_data.py

```diff
@@ -54,3 +60,81 @@ def _build_binary_string(name: str, num_bits: int) -> str:
             "special_feature_optins", SPECIAL_FEATURE_BITS
         ),
     )
+
+
+def _integer_dict_to_list(int_dict: Dict[int, bool]) -> List[int]:
+    """Converts a dictionary of integers->booleans into a list of just the integers mapped to True"""
+    return [identifier for identifier, consented in int_dict.items() if consented]
+
+
+def convert_tc_string_to_mobile_data(tc_str: Optional[str]) -> Optional[TCMobileData]:
```

Comment:
This same method is used in the new Plus endpoint that decodes a TC string into TC Mobile data format -

Reply:
thank you so much 😄